go/server.go

```diff
@@ -87,6 +87,27 @@ func getAccessToken(c *gin.Context) {
 	})
 }
 
+// Creates an item add token for initializing Link in normal item add mode.
+// In production, this endpoint should be authenticated.
+func getItemAddToken(c *gin.Context) {
+	itemAddToken := c.PostForm("item_add_token")
+	// In production, this client_user_id must be a unique identifier for each user that accesses Link.
+	clientUserID := time.Now().String()
+	response, err := client.CreateItemAddToken(plaid.ItemAddTokenUserFields{ClientUserID: clientUserID})
+
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+
+	itemAddToken = response.AddToken
```

Comment:
If the above suggestion, which removes the definition of `itemAddToken` it must be defined here.
```suggestion
	itemAddToken := response.AddToken
```